ospfd/ospfd.c

```diff
@@ -936,6 +936,14 @@ static void ospf_finish_final(struct ospf *ospf)
 	XFREE(MTYPE_OSPF_TOP, ospf);
 }
 
+static void ospf_range_table_node_destroy(route_table_delegate_t *delegate,
+			struct route_table *table, struct route_node *node)
+{
+	XFREE(MTYPE_OSPF_AREA_RANGE, node->info);
+	XFREE(MTYPE_ROUTE_NODE, node);
+}
+
+route_table_delegate_t ospf_range_table_delegate = {.create_node = route_node_create, .destroy_node = ospf_range_table_node_destroy};
```

Comment:
This seems to be "unformatted", but CI/frrbot do not complain :shrug:

Reply:
Should `new->nssa_ranges = route_table_init();` in ospfd.c:983 do the same init as new->ranges?

Reply:
Thank you for the review!
I have fixed the formatting.